### PR TITLE
Balance & Descriptions

### PR DESCRIPTION
**Pull request by l34Um1**
Also made Shivas Cuirass make sense. Point wasn't really to balance it but just to not have it become a worse item by upgrading

### CODE REVIEW

**Comment by Minnakht** [comment]
@Haganeko made satanic core changes earlier in #793.

File: game/scripts/npc/abilities/phoenix_sun_ray.txt [hunk 1]
```diff
@@ -56,9 +56,9 @@
 			"05"
 			{
 				"var_type"			"FIELD_FLOAT"
-				"hp_perc_heal"			"0.625 1.25 1.875 2.5 3.75 8.75"
+				"hp_perc_heal"			"0.625 1.25 1.875 2.5 2.5 2.5"
 			}
```
Comment (Haganeko) [inline game/scripts/npc/abilities/phoenix_sun_ray.txt]: So Sun Ray doesn't progress at all anymore? That sounds bad.

File: game/scripts/npc/items/item_lucience_3.txt [hunk 2]
```diff
@@ -42,7 +42,7 @@
     "ScriptFile"                    "items/lucience.lua"
 		"AbilityBehavior"				"DOTA_ABILITY_BEHAVIOR_NO_TARGET | DOTA_ABILITY_BEHAVIOR_TOGGLE"
 		"AbilityTextureName"            "custom/lucience_3"
-		"AbilityCooldown"               "3"
```
Comment (Haganeko) [inline game/scripts/npc/items/item_lucience_3.txt]: Does Lucience need a buff, of all things?

File: game/scripts/npc/items/item_satanic_core_3.txt [hunk 3]
```diff
@@ -98,7 +98,7 @@
 			"10"
 			{
 				"var_type"				"FIELD_FLOAT"
-				"hero_lifesteal"		"50 65 80"
```
Comment (Haganeko) [inline game/scripts/npc/items/item_satanic_core_3.txt]: I have my own review of Satanic Core pending. Would you be willing to let me have that? Or at least give it a look?

File: game/scripts/npc/items/item_shivas_cuirass.txt [hunk 4]
```diff
@@ -115,23 +115,23 @@
...
 			{
 				"var_type"				"FIELD_INTEGER"
-				"aura_attack_speed"		"-55"
+				"aura_attack_speed"		"-85 -95"
 			}
 			// Assault Cuirass Parameters
 			"11"
 			{
 				"var_type"				"FIELD_INTEGER"
-				"aura_negative_armor"	"-5"
```
Comment (Haganeko) [inline game/scripts/npc/items/item_shivas_cuirass.txt]: I have a post in balance-discussion regarding -armor values. The ones we have for every item but Crest are way undertoned. I'll be fixing this soon.

**Pull request merged**